include/deal.II/matrix_free/fe_evaluation_data.h

```diff
@@ -472,7 +472,7 @@ class FEEvaluationData
   get_cell_ids() const
   {
     // implemented inline to avoid compilation problems on Windows
-    Assert(cell != numbers::invalid_unsigned_int, ExcNotInitialized());
+    Assert(is_reinitialized, ExcNotInitialized());
```

Comment:
In this line we have a large number of matrix-free tests currently failing with a message like
```
--------------------------------------------------------
An error occurred in line <482> of file </srv/testsuite/dealii/include/deal.II/matrix_free/fe_evaluation_data.h> in function
    const std::array<unsigned int, dealii::FEEvaluationData<dim, Number, is_face>::n_lanes>& dealii::FEEvaluationData<dim, Number, is_face>::get_cell_ids() const [with int dim = 3; Number = dealii::VectorizedArray<double, 2>; bool is_face = false]
The violated condition was: 
    is_reinitialized
Additional information: 
    (none)
--------------------------------------------------------
```
The reason is that `is_reinitialized` only gets set in debug mode, but `Assert` actually maps to something non-trivial in the test suite and hence signals problems. See also the list of new failures after this commit: https://cdash.43-1.org/viewTest.php?onlydelta&buildid=9580